src/renderer/components/+extensions/extensions.tsx

```diff
@@ -32,27 +54,238 @@ export class Extensions extends React.Component {
+      Notifications.ok(
+        <p>Extension <b>{name}/{version}</b> successfully installed!</p>
+      );
+    } catch (err) {
+      Notifications.error(
+        <p>Installing extension <b>{name}</b> has failed: <em>{err}</em></p>
+      );
+    } finally {
+      // clean up
+      fse.remove(unpackingTempFolder).catch(Function);
+      fse.unlink(tmpFile).catch(Function);
+    }
+  }
+
   renderInfo() {
     return (
-      <div className="flex column gaps">
+      <div className="extensions-info flex column gaps">
         <h2>Lens Extension API</h2>
         <div>
           The Extensions API in Lens allows users to customize and enhance the Lens experience by creating their own menus or page content that is extended from the existing pages. Many of the core
           features of Lens are built as extensions and use the same Extension API.
         </div>
         <div>
-          Extensions loaded from:
-          <div className="extensions-path flex inline">
+          <p><em>Extensions loaded from:</em></p>
+          <div className="extensions-path flex inline" onClick={() => shell.openPath(this.extensionsPath)}>
+            <Icon material="folder" tooltip={{ children: "Open folder", preferredPositions: "bottom" }}/>
             <code>{this.extensionsPath}</code>
-            <Icon
-              material="folder"
-              tooltip="Open folder"
-              onClick={() => shell.openPath(this.extensionsPath)}
-            />
           </div>
         </div>
-        <div>
-          Check out documentation to <a href="https://docs.k8slens.dev/" target="_blank">learn more</a>
+        <div className="install-extension flex column gaps">
+          <em>
+            Install extensions from tarball ({this.supportedFormats.join(", ")}):
+          </em>
+          <Input
+            showErrorsAsTooltip={true}
+            className="box grow"
+            theme="round-black"
+            placeholder="URL or npm-package-name"
+            value={this.downloadUrl}
+            onChange={v => this.downloadUrl = v}
+            onSubmit={this.installExtensions}
+          />
+          <Button
+            primary
+            label="Add extensions"
+            onClick={this.installExtensions}
+          />
+          <p className="hint">
+            <Trans><b>Pro-Tip 1</b>: you can download packed extension from NPM via</Trans>
+            <Clipboard showNotification>
+              <code>npm pack %package-name</code>
+            </Clipboard>
+          </p>
+          <p className="hint">
+            <Trans><b>Pro-Tip 2</b>: you can drag & drop extension's tarball here to request installation</Trans>
+          </p>
+        </div>
+        <div className="more-info flex inline gaps align-center">
+          <Icon material="local_fire_department"/>
+          <p>
+            Check out documentation to <a href="https://docs.k8slens.dev/" target="_blank">learn more</a>
```

Comment:
This url is referenced in a few places in our codebase. We should add it as a constant.